backend/src/test/java/com/redhat/cloud/notifications/db/repositories/SubscriptionRepositoryTest.java

```diff
@@ -108,13 +121,19 @@ void testFindEmailSubscriptionsByUserId() {
         final String userIdTwo = "userIdTwo";
 
         final Random random = new Random();
-        this.subscriptionRepository.updateSubscription(DEFAULT_ORG_ID, userIdOne, randomEventType.getId(), SubscriptionType.INSTANT, random.nextBoolean());
-        this.subscriptionRepository.updateSubscription(DEFAULT_ORG_ID, userIdOne, randomEventTypeTwo.getId(), SubscriptionType.INSTANT, random.nextBoolean());
-        this.subscriptionRepository.updateSubscription(DEFAULT_ORG_ID, userIdOne, randomEventTypeThree.getId(), SubscriptionType.INSTANT, random.nextBoolean());
-
-        this.subscriptionRepository.updateSubscription(DEFAULT_ORG_ID, userIdTwo, randomEventType.getId(), SubscriptionType.INSTANT, random.nextBoolean());
-        this.subscriptionRepository.updateSubscription(DEFAULT_ORG_ID, userIdTwo, randomEventTypeTwo.getId(), SubscriptionType.INSTANT, random.nextBoolean());
-        this.subscriptionRepository.updateSubscription(DEFAULT_ORG_ID, userIdTwo, randomEventTypeThree.getId(), SubscriptionType.INSTANT, random.nextBoolean());
+        boolean subscribed = random.nextBoolean();
+        this.subscriptionRepository.updateSubscription(DEFAULT_ORG_ID, userIdOne, randomEventType.getId(), SubscriptionType.INSTANT, random.nextBoolean(), buildAllSeveritiesUpdateDetails(subscribed));
```

Comment:
Shouldn't we do this instead?

```suggestion
        this.subscriptionRepository.updateSubscription(DEFAULT_ORG_ID, userIdOne, randomEventType.getId(), SubscriptionType.INSTANT, subscribed, buildAllSeveritiesUpdateDetails(subscribed));
```
Otherwise, the `email_subscriptions` record could have `subscribed=false` while severities might be subscribed, which looks inconsistent.